src/Form/Number.php

```diff
@@ -4,27 +4,7 @@
 
 namespace MyParcelCom\JsonSchema\FormBuilder\Form;
 
-use MyParcelCom\JsonSchema\FormBuilder\Properties\JsonSchemaProperty;
-use MyParcelCom\JsonSchema\FormBuilder\Properties\PropertyType;
-
-class Number implements Field
+class Number extends AbstractField
 {
-    public function __construct(
-        public readonly string $name,
-        public readonly string $label,
-        public readonly bool $isRequired = false,
-        public readonly ?string $help = null,
-    ) {
-    }
-
-    public function toJsonSchemaProperty(): JsonSchemaProperty
-    {
-        return new JsonSchemaProperty(
-            name: $this->name,
-            type: PropertyType::NUMBER,
-            description: $this->label,
-            isRequired: $this->isRequired,
-            help: $this->help,
-        );
-    }
+    protected FieldType $fieldType = FieldType::NUMBER;
```

Comment:
Here we can apply the abstract protected method idea too - forcing all implementing children to supply a fieldType and thus avoiding uninitialized property exceptions.